policy-controller/k8s/status/src/routes/http.rs

```diff
@@ -1,30 +1,5 @@
-use crate::resource_id::ResourceId;
-use gateway::{CommonRouteSpec, HttpBackendRef};
-use linkerd_policy_controller_k8s_api::{
-    gateway,
-    policy::{self, Server},
-    Service,
-};
-
-/// Represents an HTTPRoute's parent reference from its spec.
-///
-/// This is separate from the policy controller index's `InboundParentRef`
-/// because it does not validate that the parent reference is not in another
-/// namespace. This is something that should be relaxed in the future in the
-/// policy controller's index and we could then consider consolidating these
-/// types into a single shared lib.
-#[derive(Clone, Eq, PartialEq)]
-pub enum ParentReference {
-    Server(ResourceId),
-    Service(ResourceId, Option<u16>),
-    UnknownKind,
-}
-
-#[derive(Clone, Eq, PartialEq)]
-pub enum BackendReference {
-    Service(ResourceId),
-    Unknown,
-}
+use super::{BackendReference, ParentReference};
+use linkerd_policy_controller_k8s_api::gateway::{CommonRouteSpec, HttpBackendRef};
 
 pub(crate) fn make_parents(namespace: &str, route: &CommonRouteSpec) -> Vec<ParentReference> {
```

Comment:
Any reason this is defined in http.rs and then re-exported in grpc.rs instead of just being in mod.rs?